insteon_mqtt/device/Dimmer.py

```diff
@@ -465,6 +465,69 @@ def set_backlight(self, level, on_done=None):
         msg_handler = handler.StandardCmd(msg, self.handle_backlight, on_done)
         self.send(msg, msg_handler)
 
+    #-----------------------------------------------------------------------
+    def get_flags(self, on_done=None):
+        """Hijack base get_flags to inject extended flags request.
+
+        The flags will be passed to the on_done callback as the data field.
+        Derived types may do something with the flags by override the
+        handle_flags method.
+
+        Args:
+          on_done: Finished callback.  This is called when the command has
+                   completed.  Signature is: on_done(success, msg, data)
+        """
+        seq = CommandSeq(self, "Dimmer get_flags complete", on_done,
+                         name="GetFlags")
+        seq.add(super().get_flags)
+        seq.add(self._get_ext_flags)
+        seq.run()
+
+    #-----------------------------------------------------------------------
+    def _get_ext_flags(self, on_done=None):
+        """Get the Insteon operational extended flags field from the device.
+
+        For the dimmer device, the flags include on-level and ramp-rate.
+
+        Args:
+          on_done: Finished callback.  This is called when the command has
+                   completed.  Signature is: on_done(success, msg, data)
+        """
+        LOG.info("Dimmer %s cmd: get extended operation flags", self.label)
+
+        # Requesting data d1 = 0x01
```

Comment:
There's nothing wrong with the code here, but maybe the comment is inaccurate?  Doesn't D1 contain the group # (1) and D2 =0x00 is what indicates that we want a data request?

(If you agree with this change, then the same should be made to insteon_mqtt/device/KeypadLinc.py.)

Reply:
D1 could be the group number, that is what the KPL does.  

D2-D13 are 0x00.  The documentation doesn't describe what if anything D2 is.  You are correct that responses in this case return 0x01.  But I would avoid suggesting that this is any type of standard across devices.

Reply:
Oops, you are correct.  It looks like D9 may be the LED light level.

Reply:
I'll admit, my first source of information was just [this page](http://www.madreporite.com/insteon/commands.htm), which isn't exactly official.

Upon further digging, I found something similar in the Insteon Dev Guide.  I'm not sure if this is the latest version, but I see on page 145/146 of the [Aug 16, 2007 version of the Insteon Developer's Guide](https://cache.insteon.com/pdf/INSTEON_Developers_Guide_20070816a.pdf) that D2 = 0x00 means "Data Request" for Dimmers.  It says something similar for KeypadLinc's on page 143.